docker-compose.yml

```diff
@@ -10,6 +10,7 @@ services:
       - ./plugins/events:/var/www/html/admin/tool/events
       - ./plugins/study:/var/www/html/mod/study
       - ./plugins/content:/var/www/html/filter/content
+      - ./plugins/direct_events_plugin:/var/www/html/local/direct
       - .:/repo
```

Comment:
I couldn't put this comment on the actual file, so making it here to track: Let's add some content to the `README.md` to help our future selves remember what this plugin is for / doing.